src/main/java/com/opencart/datamodel/ProductModel.java

```diff
@@ -0,0 +1,21 @@
+package com.opencart.datamodel;
+
+import java.util.*;
+
+public class ProductModel {
```

Comment:
Could you, please, the value of this class, and why do you need it?

Reply:
I wanted to collect somewhere valid Prices of Products that are on the main page and have the possibility to call for valid Prices using ProductName. 
This class contains a method that returns valid ProductPrice using the parameter ProductName. 
I am not sure about this class and if collection HashMap is the right collection to use. Maybe not HashMap but ArrayList should be used to store Products Names and Prices?